### PR TITLE
OI4-205

### PR DESCRIPTION
**Pull request by Weinschenk**

### CODE REVIEW

File: packages/oi4-oec-service-opcua-model/src/index.ts [hunk 1]
```diff
@@ -67,6 +68,50 @@ interface IOPCUABuilderProps {
   max: number;
 }
 
+export const jsonSchemaBuilder = () => {
```
Comment (vdedominicis-EH) [inline packages/oi4-oec-service-opcua-model/src/index.ts]: What about exporting this function into a separate helper or utils file? It has no dependency from the rest of the code and it is pretty big.

**Review by vdedominicis-EH — CHANGES_REQUESTED** [review]
All good :) There's just the tiny change I marked as code change but beside this looks fine.

File: packages/oi4-oec-service-opcua-model/src/index.ts (hunk 1 above)
Comment (Weinschenk) [reply packages/oi4-oec-service-opcua-model/src/index.ts]: Valid point. Refactored the builder to be part of the OpcUaSchemaProvider.ts and OecSchemaProvider.ts. Furthermore I removed the duplicate code.

Please give it another try...

**Review by vdedominicis-EH — APPROVED** [review]
looks good :)

**Pull request merged**